### ISSUE TITLE
toobig

### ISSUE THREAD

**Issue by ibuilder** [TiagoSilvaPereira/vemto-issues#2]
```
RangeError: Maximum call stack size exceeded
    at d.belongsTo (app://./js/chunk-common.56fadd40.js:1:42649)
    at app://./js/generator.e9187e3e.js:1:31109
    at Array.forEach (<anonymous>)
    at Ee.calculateEntityWeight (app://./js/generator.e9187e3e.js:1:31095)
    at app://./js/generator.e9187e3e.js:1:31134
    at Array.forEach (<anonymous>)
    at Ee.calculateEntityWeight (app://./js/generator.e9187e3e.js:1:31095)
    at app://./js/generator.e9187e3e.js:1:31134
    at Array.forEach (<anonymous>)
    at Ee.calculateEntityWeight (app://./js/generator.e9187e3e.js:1:31095)
```

---

vemto 0.4.10
Electron 8.5.0
win32 10.0.18362
Locale: en-US

**Comment by TiagoSilvaPereira** [comment]
Hi, @ibuilder thanks for reporting this issue. Could you please send a PrintScreen of the schema so I can reproduce the issue?

**Comment by TiagoSilvaPereira** [comment]
@ibuilder I just noticed you are on 0.4.10, this bug was fixed on 0.4.12... please update to the latest version to test again (by default, Vemto updates automatically after closing and opening again, but you can download the latest version [here](https://github.com/TiagoSilvaPereira/vemto-releases/releases)). Thanks
